beetmoverscript/zip.py

```diff
@@ -129,7 +129,7 @@ def _fetch_zip_metadata(zip_file):
             'compress_size': info.compress_size,
             'file_size': info.file_size,
         }
-        for info in zip_file.infolist()
+        for info in zip_file.infolist() if not info.is_dir()
```

Comment:
Nit: I'd add a TODO to say: "Write a check to ensure no empty directory was created"

Reply:
Done.